src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -1,393 +1,39 @@
-    - purchases_and_sales_of_ancillary_services_398_duration
-    - purchases_and_sales_of_ancillary_services_398_instant
-    - monthly_transmission_system_peak_load_400_duration
-    - monthly_transmission_system_peak_load_400_instant
-    - monthly_iso_rto_transmission_system_peak_load_400a_duration
-    - monthly_iso_rto_transmission_system_peak_load_400a_instant
-    - electric_energy_account_401a_duration
-    - electric_energy_account_401a_instant
-    - monthly_peaks_and_output_401b_duration
-    - monthly_peaks_and_output_401b_instant
-    - steam_electric_generating_plant_statistics_large_plants_402_duration
-    - steam_electric_generating_plant_statistics_large_plants_402_instant
-    - steam_electric_generating_plant_statistics_large_plants_fuel_statistics_402_duration
-    - steam_electric_generating_plant_statistics_large_plants_fuel_statistics_402_instant
-    - hydroelectric_generating_plant_statistics_large_plants_406_duration
-    - hydroelectric_generating_plant_statistics_large_plants_406_instant
-    - pumped_storage_generating_plant_statistics_large_plants_408_duration
-    - pumped_storage_generating_plant_statistics_large_plants_408_instant
-    - generating_plant_statistics_410_duration
-    - generating_plant_statistics_410_instant
-    - energy_storage_operations_large_plants_414_duration
-    - energy_storage_operations_large_plants_414_instant
-    - energy_storage_operations_large_plants_totals_414_duration
-    - energy_storage_operations_large_plants_totals_414_instant
-    - transmission_line_statistics_422_duration
-    - transmission_line_statistics_422_instant
-    - transmission_line_statistics_totals_422_duration
-    - transmission_line_statistics_totals_422_instant
-    - transmission_lines_added_during_year_424_duration
-    - transmission_lines_added_during_year_424_instant
-    - transmission_lines_added_during_year_total_424_duration
-    - transmission_lines_added_during_year_total_424_instant
-    - substations_426_duration
-    - substations_426_instant
-    - substations_total_426_duration
-    - substations_total_426_instant
-    - transactions_with_associated_affiliated_companies_provided_by_429_duration
-    - transactions_with_associated_affiliated_companies_provided_by_429_instant
-    - transactions_with_associated_affiliated_companies_provided_for_429_duration
-    - transactions_with_associated_affiliated_companies_provided_for_429_instant
+  ferc1_xbrl_to_sqlite_settings:
+    years: [2021]
+    # A list of tables to be loaded into the local SQLite database. These are
+    # the table names as created from the 2022 XBRL taxonomy.
+    tables:
+      - identification_001_duration
+      - steam_electric_generating_plant_statistics_large_plants_fuel_statistics_402_duration
+
+  ferc2_xbrl_to_sqlite_settings:
+    years: [2021]
+  ferc6_xbrl_to_sqlite_settings:
+    years: [2021]
+  ferc60_xbrl_to_sqlite_settings:
+    years: [2021]
+  ferc714_xbrl_to_sqlite_settings:
+    years: [2021]
```

Comment:
What happens when no tables are given? Does it just get everything? I think it would be more readable to have a special value "all" or something that's translated into the list of all the tables by the Pydantic validator (as we do with the EPA CEMS for the states)

Reply:
Yeah I think an "all" keyword would be really nice to have for FERC 1. For these forms, however, I haven't even included `tables` as a parameter in their settings models, because I don't really see a use case for extracting only a subset of tables. If we're only using these forms to publish the raw SQLite DB's I feel like we'll always want all of the tables

Reply:
Okay, fair enough. We can add it later if there's some more specific testing / ETL application.

If you feel like adding a special "all" keyword for the FERC1 it'll make the settings files much easier to manage probably. But it'd be good to have it be the same for the DBF and the XBRL sides. If that feels out of scope don't worry about it!